Documentation/Contributing/development-flow.md

```diff
@@ -256,119 +242,101 @@ Common cases that may need tests:
+2. Add the backport label to that PR such as backport-release-1.11
+3. After the PR is merged to master, the `mergify` bot will automatically open a PR with the commits backported to the release branch
+4. After the CI is green and a maintainer has approved the PR, the bot will automatically merge the backport PR
 
-## Testing changes and debugging issues in Ceph manager modules
+## Debugging issues in Ceph manager modules
 
 The Ceph manager modules are written in Python and can be individually and dynamically loaded from the manager. We can take advantage of this feature in order to test changes and to debug issues in the modules.
-This is just a hack to test/debug quickly and easily any modification in the manager modules.
+This is just a hack to debug any modification in the manager modules.
 
-The ceph dashboard and the rook orchestrator modules are probably the two modules more prone to have modifications affecting the rook cluster, and therefore it is interesting to know how to debug and test changes easily in these modules.
+The `dashboard` and the `rook` orchestrator modules are the two modules most commonly have modifications that need to be tested.
 
-The way to proceed is to make the modification directly in the manager module and reload it:
+Make modifications directly in the manager module and reload:
 
 1. Update the cluster so only a single mgr pod is running. Set the `mgr.count: 1` in the CephCluster CR if it is not already.
 
-Now shell into the manager container:
-```console
-kubectl exec -n rook-ceph --stdin --tty $(kubectl get pod -n rook-ceph -l ceph_daemon_type=mgr,instance=a  -o jsonpath='{.items[0].metadata.name}') -c mgr  -- /bin/bash
-```
+2. Shell into the manager container:
 
-2. Make the modifications needed in the required manager module.
-
-The source code of all the manager modules is in:
 ```console
-/usr/share/ceph/mgr/
+kubectl exec -n rook-ceph --stdin --tty $(kubectl get pod -n rook-ceph -l ceph_daemon_type=mgr,instance=a  -o jsonpath='{.items[0].metadata.name}') -c mgr  -- /bin/bash
 ```
 
-Enter the folder of the manager module to make your changes.
+3. Make the modifications needed in the required manager module. The manager module source code is found in `/usr/share/ceph/mgr/`.
 
 !!! Note
-    If the manager pod is restarted for whatever reason, all the modifications made in the mgr container will be lost.
+    If the manager pod is restarted, all modifications made in the mgr container will be lost
 
-3. Restart the manager module modified to test/debug the modifications:
+1. Restart the modified manager module to test the modifications:
 
 Example for restarting the rook manager module with the [krew plugin](https://github.com/rook/kubectl-rook-ceph):
 ```console
 kubectl rook-ceph ceph mgr module disable rook
 kubectl rook-ceph ceph mgr module enable rook
 ```
 
-Once the module is restarted the modifications will be running in the active manager, and you will be able to test/debug them.  
```

Comment:
I think with big sentence it gives more clearity

Reply:
Which phrase exactly? The way to test or debug will depend on the change, so I wasn't sure the long message helps.